pkg/db-repo/orm.go

```diff
@@ -23,14 +24,21 @@ type OrmSettings struct {
 	Application string              `cfg:"application" default:"{app_name}"`
 }
 
-func NewOrm(ctx context.Context, config cfg.Config, logger log.Logger) (*gorm.DB, error) {
-	connection, err := db.ProvideConnection(ctx, config, logger, "default")
-	if err != nil {
+func NewOrm(ctx context.Context, config cfg.Config, logger log.Logger, dbClientName string) (*gorm.DB, error) {
+	var err error
+	var connection *sqlx.DB
+
+	if dbClientName == "" {
```

Comment:
I believe, for other entities we don't have this fallback. As it's more explicit, I suggest to pass in "default" at the callee site instead of an empty string.